src/main/java/com/google/codeu/data/Datastore.java

```diff
@@ -79,4 +79,37 @@ public void storeMessage(Message message) {
 
     return messages;
   }
+
+  /**
+  * Stores the User in Datastore Entity
+  */
+ public void storeUser(User user) {
+  Entity userEntity = new Entity("User", user.getEmail());
+  userEntity.setProperty("email", user.getEmail());
+  userEntity.setProperty("aboutMe", user.getAboutMe());
+  datastore.put(userEntity);
+ }
+
+ /**
+  * Returns the User of email address with aboutMe, or
+  * null if no matching User was found.
+  */
+ public User getUser(String email) {
+  Query query = new Query("User") .setFilter(new Query.FilterPredicate(
+                                                        "email",
+                                                        FilterOperator.EQUAL,
+                                                        email
+                                                      )
+                                            );
+  PreparedQuery results = datastore.prepare(query);
+  Entity userEntity = results.asSingleEntity();
+  if(userEntity == null) {
+   return null;
+  }
+
+  //Return aboutMe
+  String aboutMe = (String) userEntity.getProperty("aboutMe");
+  User user = new User(email, aboutMe);
```

Comment:
It might be better to use userEntity.getProperty("email") here instead of email. My main thinking here is that doing that would be consistent with what you do in storeUser(). It would also make this code a little more robust if you changed the query filter to be case-insensitive.

Reply:
Implemented in commit 2b653b685d4a583914bca81d3cbdaf3ca5db695a